images/kubectl-build-deploy-dind/helmcharts/postgres-dbaas/templates/prebackuppod.yaml

```diff
@@ -0,0 +1,52 @@
+{{ if .Capabilities.APIVersions.Has "backup.appuio.ch/v1alpha1/PreBackupPod" }}
+apiVersion: backup.appuio.ch/v1alpha1
+kind: PreBackupPod
+metadata:
+  name: {{ include "postgres-dbaas.fullname" . }}-prebackuppod
+  labels:
+    {{- include "postgres-dbaas.labels" . | nindent 4 }}
+  annotations:
+    {{- include "postgres-dbaas.annotations" . | nindent 4 }}
+spec:
+  backupCommand: /bin/sh -c "PGPASSWORD=$BACKUP_DB_PASSWORD pg_dump --host=$BACKUP_DB_HOST --port=$BACKUP_DB_PORT --dbname=$BACKUP_DB_NAME -w"
+  fileExtension: .{{ include "postgres-dbaas.fullname" . }}.tar
```

Comment:
is the `pg_dump` really returning a `tar` file? I assume this is probably a textfile?

Reply:
Nope, this command is dumping its output directly to stdout, as required by k8up :) https://k8up.io/k8up/1.0.0/references/object-specifications.html#_settings_7

Reply:
I suppose we could label it as an sql file instead of a tar file, though; that would make much more sense

Reply:
yes, correct, you need to tell k8up which file extension that should be used to save the file in.

Reply:
I read into this, and it looks like we can actually get tar formatted data from pg_dump. Not only can we, it's the most portable format according to their docs. :)